resources/css/style.css

```diff
@@ -223,19 +227,19 @@ textarea.materialize-textarea {
 }
 :-moz-placeholder {
   /* Firefox 18- */
-
+  text-align: center;
```

Comment:
I don't think we need `text-align: center;` for placeholder It works without it :+1: